Porting/release_managers_guide.pod

```diff
@@ -981,9 +981,9 @@ Then delete the temporary installation.
 
 =head3 Create the release tag
 
-Create the I<annotated> tag identifying this release (e.g.):
+Create the I<annotated> tag identifying this release:
 
- $ git tag v5.21.4 -m 'Perl 5.21.4'
+ $ git tag v5.X.Y -m 'Perl 5.X.Y'
```

Comment:
Are we ever showing the output of those commands in the RMG?

If not, leaving the prompt out makes them easier to copy/paste, I think.

Reply:
> Are we ever showing the output of those commands in the RMG?

Yes

> If not, leaving the prompt out makes them easier to copy/paste, I think.

Here again, I followed the file convention. And I think I told myself the same, tried, and keeping the prompt appeared nicer (but slightly less handy I agree)